### PR TITLE
Classes lesson : Add one knowledge check

### PR DESCRIPTION
**Pull request by Tribhuwan-Joshi**
Knowledge check about static variables and properties  in class

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

**Comment by 01zulfi** [comment]
- Putting this on hold until https://github.com/TheOdinProject/curriculum/pull/24432 is reviewed.

**Comment by Tribhuwan-Joshi** [comment]
@01zulfi  can u review this pr now

**Review by 01zulfi — CHANGES_REQUESTED** [review]
The change looks good, but unfortunately there's a merge conflict that needs to be fixed.

File: javascript/organizing_your_javascript_code/classes.md [hunk 1]
```diff
@@ -60,5 +60,8 @@ This section contains questions for you to check your understanding of this less
 * <a class="knowledge-check-link" href="https://javascript.info/property-accessors">What are "getters" & "setters"?</a>
 * <a class="knowledge-check-link" href="https://javascript.info/class">Describe computed names and class fields.</a>
 * <a class="knowledge-check-link" href="https://javascript.info/class">Describe function binding.</a>
+* <a class="knowledge-check-link" href="(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes)">Describe static properties.</a>
+* <a class="knowledge-check-link" href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes">Be able to use inheritance with classes.</a>
+* <a class="knowledge-check-link" href="https://rajaraodv.medium.com/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65">Briefly talk about the conflict in JS with functional programming and classes.</a>
```
Comment (01zulfi) [inline javascript/organizing_your_javascript_code/classes.md:65]: These two knowledge checks were removed in another PR https://github.com/TheOdinProject/curriculum/pull/24432/files. I don't think we need to add them back.

**Review by 01zulfi — CHANGES_REQUESTED** [review]

**Review by 01zulfi — APPROVED** [review]
Thanks for making this change and resolving the merge conflicts!

**Pull request merged**